client/landing/stepper/declarative-flow/internals/steps-repository/bundle-transfer/index.tsx

```diff
@@ -119,7 +120,11 @@ const BundleTransfer: Step = function BundleTransfer( { navigation } ) {
 			// Ensure we don't have an existing transfer in progress before starting a new one.
 			if ( preTransferCheck?.status !== transferStates.ACTIVE ) {
 				// Initiate transfer
-				await initiateAtomicTransfer( siteId, softwareSet );
+				if ( isNewSiteMigrationFlow( flow ) ) {
+					await initiateAtomicTransfer( siteId, softwareSet, 'migrate' );
+				} else {
+					await initiateAtomicTransfer( siteId, softwareSet );
+				}
```

Comment:
Non-blocking: I dislike this kind of duplication. Might it be cleaner to default a `transferIntent` local variable to `undefined` and then override when we know it should be a value?
```suggestion
				const transferIntent = isNewSiteMigrationFlow( flow ) ? 'migrate' : undefined;

				await initiateAtomicTransfer( siteId, softwareSet, transferIntent );
```

Reply:
yes, prettier indeed. applied!